src/core_modules/capture-core/components/Pages/ViewEvent/EventDetailsSection/EventDetailsSection.component.js

```diff
@@ -111,11 +114,33 @@ class EventDetailsSectionPlain extends Component<Props> {
     render() {
         const {
             classes,
+            programStage: { name, icon },
+            hasName,
```

Comment:
I need to display the `name` and the `icon` only in a specific situation. Therefore I cannot check if they are null/undefined because they almost always have a value.

Reply:
In that case, I would rename the variable to something as `shouldShowIcon` or `shouldShowName` as you said the value always present in most case ( `hasIcon` makes me think we want to check the value present). WDYT?